### PR TITLE
mark columns as summable

### PR DESCRIPTION
**Pull request by linki**
this pull request allows to globally define which columns can be summed up

caveat:

it's still possible in the admin interface to mark columns that cannot or shouldn't be summed up (like string columns or foreign keys)

in the end, only the columns that are marked and can be summed up will be summed up..

### CODE REVIEW

File: app/helpers/settings_helper.rb [hunk 1]
```diff
@@ -49,6 +49,30 @@ def setting_multiselect(setting, choices, options={})
...
+            '<td>' + h(text) + '</td>' +
+            settings.collect do |setting|
+              '<td align="center">' + check_box_tag("settings[#{setting}][]", value, Setting.send(setting).include?(value)) + '</td>'
+            end.join +
+          '</tr>'
+        end.join +
+      '</tbody>' +
+    '</table>'
+  end
+
```
Comment (linki) [inline app/helpers/settings_helper.rb:75]: tried to implement this helper with content_tags, but the result was even worse

**Comment by ulferts** [comment]
I like the code and currently see no way around this ugly concatenation of dom-strings.

Just two remarks:
- an integration test should test that a column marked as summable is actually summed up
- when no column in marked as summable, the check box where the user can choose to sum up columns in an issue report should not be presented/disabled.

**Comment by linki** [comment]
done + tests in plugin

**Comment by linki** [comment]
the tests for this still fail on the ci because they are in the chiliproject_tests plugin (branch feature/list_sums_configuration)

could someone please review and merge this PR?

**Pull request merged**